### PR TITLE
Contains DrugGroup class, Service, ServiceImpl, DAO, HibernateDAO, JUnit tests, updated liquibase, etc.

### PR DESCRIPTION
**Pull request by diegodayan**
Added: DrugGroup.java
Added: DrugGroupService.java
Added: DrugGroupServiceImpl.java
Added: DrugGroupDAO.java
Added: HibernateDrugGroupDAO.java
Added: RaxacoreDrugGroup.hbm.xml
Added: DrugGroupController.java
Added: DrugGroupResource.java
Added: HibernateDrugGroupDAOTest.java
Added: DrugGroupServiceImplTest.java
Changed: liquibase.xml
Changed: moduleTestData.xml
Changed: moduleApplicationContext.xml
Changed: config.xml
Changed: TestingApplicationContext.xml

### CODE REVIEW

File: omod/src/main/java/org/raxa/module/raxacore/web/v1_0/resource/DrugGroupResource.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+import org.openmrs.module.webservices.rest.web.resource.impl.NeedsPaging;
+import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+import org.raxa.module.raxacore.DrugGroup;
+import org.raxa.module.raxacore.DrugGroupService;
+
+/**
+ * {@link Resource} for DrugGroup, supporting standard CRUD operations This
+ * resource is currently not used because of serialization issue in OpenMRS core
+ * (TRUNK-2205)
```
Comment (johnstoecker) [inline omod/src/main/java/org/raxa/module/raxacore/web/v1_0/resource/DrugGroupResource.java:40]: Yes -- I ran into this issue as well. Have emailed the OpenMRS dev list to see if there is any way we can get around it.

File: api/src/main/java/org/raxa/module/raxacore/impl/DrugGroupServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+import org.openmrs.api.context.Context;
+import org.raxa.module.raxacore.DrugGroup;
+import org.raxa.module.raxacore.DrugGroupService;
+import org.raxa.module.raxacore.db.DrugGroupDAO;
+
+/*
+ * Implements PatientListService.java Note the PatientList query must be in the
+ * form of:
+ * "?encounterType=<uuid>&startDate=2012-05-07&endDate=2012-05-08&inlist=<uuidForList>&notinlist=<uuidForList>"
+ */
```
Comment (johnstoecker) [inline api/src/main/java/org/raxa/module/raxacore/impl/DrugGroupServiceImpl.java]: Could be confusing keeping this comment here

Comment (diegodayan) [inline api/src/main/java/org/raxa/module/raxacore/impl/DrugGroupServiceImpl.java]: Thanks, fixed!

**Comment by johnstoecker** [comment]
Looks good, merging.

**Pull request merged**